pkg/agent/hypershift.go

```diff
@@ -370,3 +372,33 @@ func (c *agentController) deploymentExist(ctx context.Context) (error, bool) {
 
 	return nil, true
 }
+
+func (c *agentController) readInDownstreamOverride() (*os.File, error) {
+	cm := &corev1.ConfigMap{}
+	cmKey := types.NamespacedName{Name: util.HypershiftDownstreamOverride, Namespace: c.addonNamespace}
```

Comment:
here read the cm from the addonNamespace, so does the hypershift-addon-manager need to copy the cm for **every** hypershift addon?

Reply:
the cm is deployed to the managedcluster via manifestwork.

Over here the code is reading the configmap from its own cluster